### ISSUE TITLE
how do you get the randomizer to work after you have done the steps 

### ISSUE THREAD

**Issue by warmech15** [Truthkey/KH2FMRandomizer#23]
after doing the steps layed out in the instructions i started the game but nothing was randomized, it was like i was playing normal defualt KH2FM.

**Comment by Truthkey** [comment]
Yeah, that was a problem with the Toolkit I use in conjuntion with the randomizer. It should be fixed now! Sorry for the inconveniences.
